s3po/backends/s3.py

```diff
@@ -26,7 +26,7 @@ def download(self, bucket, key, destination, retries, headers=None):
         bucket = self.get_bucket(bucket)
         # Make a file that we'll write into
         destination = CountFile(destination)
-        obj = bucket.get_key(key)
+        obj = bucket.get_key(key, validate=False)
```

Comment:
It does seem to always return a key (though the documentation doesn't expressly say so): https://github.com/boto/boto/blob/d9e5cfe900e1a58717e393c76a6e3580305f217a/boto/s3/bucket.py#L175